client/blocks/eligibility-warnings/index.jsx

```diff
@@ -2,144 +2,100 @@
+import { getSelectedSite } from 'state/ui/selectors';
+import Banner from 'components/banner';
 import Button from 'components/button';
 import Card from 'components/card';
-import { getEligibility, isEligibleForAutomatedTransfer } from 'state/automated-transfer/selectors';
-import { getSelectedSiteId } from 'state/ui/selectors';
-import SectionHeader from 'components/section-header';
+import HoldList from './hold-list';
 import QueryEligibility from 'components/data/query-atat-eligibility';
+import WarningList from './warning-list';
 
-// Mapping eligibility holds to messages that will be shown to the user
-// TODO: update supportUrls and maybe create similar mapping for warnings
-function getHoldMessages( translate ) {
-	return {
-		PLACEHOLDER: {
-			title: '',
-			description: '',
-			supportUrl: '',
-		},
-		NO_BUSINESS_PLAN: {
-			title: translate( 'Business plan required' ),
-			description: translate( 'This feature is only allowed on sites with a business plan.' ),
-			supportUrl: 'https://support.wordpress.com/'
-		},
-		NO_VIP_SITES: {
-			title: translate( 'VIP site' ),
-			description: translate( 'This feature is not supported on VIP sites.' ),
-			supportUrl: 'https://support.wordpress.com/'
-		},
-		NO_WPCOM_NAMESERVERS: {
-			title: translate( 'No WordPress.com name servers' ),
-			description: translate( 'Your custom domain must point to WordPress.com name servers.' ),
-			supportUrl: 'https://support.wordpress.com/'
-		},
-		NON_ADMIN_USER: {
-			title: translate( 'Admin access required' ),
-			description: translate( 'Only site administrators are allowed to use this feature.' ),
-			supportUrl: 'https://support.wordpress.com/'
-		},
-		NOT_USING_CUSTOM_DOMAIN: {
-			title: translate( 'No custom domain' ),
-			description: translate( 'Your site must use a custom domain to use this feature.' ),
-			supportUrl: 'https://support.wordpress.com/'
-		},
-		NOT_DOMAIN_OWNER: {
-			title: translate( 'Not a custom domain owner' ),
-			description: translate( 'You must be the owner of the primary domain subscription to use this feature.' ),
-			supportUrl: 'https://support.wordpress.com/'
-		},
-		SITE_GRAYLISTED: {
-			title: translate( 'Flagged site' ),
-			description: translate( 'This feature is not supported on sites that are not in good standing.' ),
-			supportUrl: 'https://support.wordpress.com/'
-		},
-		SITE_PRIVATE: {
-			title: translate( 'Private site' ),
-			description: translate( 'This feature is not supported on private sites.' ),
-			supportUrl: 'https://support.wordpress.com/'
-		}
-	};
-}
-
-const EligibilityWarnings = props => {
-	const {
-		translate,
-		backUrl,
-		onProceed,
-		siteId,
-		isEligible,
-		eligibilityData,
-		isPlaceholder,
-	} = props;
+export const EligibilityWarnings = ( {
+	backUrl,
+	eligibilityData,
+	hasBusinessPlan,
+	isEligible,
+	isJetpack,
+	isPlaceholder,
+	onProceed,
+	siteId,
+	siteSlug,
+	translate,
+} ) => {
+	const context = -1 !== backUrl.indexOf( 'plugins' ) ? 'plugins' : 'themes';
```

Comment:
please feel free to use `lodash.includes()` here to better indicate the goal: "does `backUrl` contain the string 'plugins'